spec/binaryTree_test.js

```diff
@@ -0,0 +1,66 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import BinaryTree from '../src/binaryTree'
+
+chai.use(chaiChange)
+
+describe('binaryTree', () => {
+  it('exists', () => {
+    expect(BinaryTree).to.be.a('function')
+  })
+
+  context('insert()', () => {
```

Comment:
I did do a white board mockup of how I was going to go about my insertions in order to test each method.